packages/wallets/package.json

```diff
@@ -6,20 +6,24 @@
     "dependencies": {
         "@deriv/api": "^1.0.0",
         "@deriv/utils": "^1.0.0",
+        "classnames": "^2.2.6",
         "embla-carousel-react": "^8.0.0-rc12",
         "formik": "^2.1.4",
         "qrcode.react": "^1.0.0",
         "react-dom": "^17.0.2",
         "react-router-dom": "^5.2.0",
         "react": "^17.0.2",
+        "react-router-dom": "^5.2.0",
```

Comment:
Is this a problem with the diff, or is it in twice?
![Uploading Screenshot 2023-09-27 at 09.32.07.png…]()

Reply:
ooof lemme check on this